### PR TITLE
Resolved Cropped text issue in Gamification Board which fixes #389

### LINKED ISSUE #389: Cropped text in Gamification Board page
## To Reproduce

Steps to reproduce the behaviour:

1. open Gamification page of the application

## Expected behavior

The text should not be cropped

## Logs

If applicable, add logs to help explain your problem (e.g. the error message and/or exception traceback).

## Environment (please complete the following information):

- OS: Linux
- Browser: Microsoft Edge

## Additional context

image

### PR DESCRIPTION
**Pull request by abhishek2x**
**Issue Number**

fixes #389

**Describe the changes you've made**

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

**Additional context (OPTIONAL)**

**Test plan (OPTIONAL)**

A good test plan should give instructions that someone else can easily follow.

**Checklist**

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] The title of my pull request is a short description of the requested changes.

**Provide a Deployed link of route/page that needs to review**
Preview: Deploy preview link here with the appropriate route

<!-- Please add deployed link to here text >
image

### CODE REVIEW

File: src/components/GamificationComponent/StatsCardSection.js [hunk 1]
```diff
@@ -58,15 +58,16 @@ const useStyles = makeStyles((theme) => ({
...
       flexWrap: "wrap",
     },
   },
   mediaCard: {
     width: 300,
+    minWidth: 165,
     boxShadow: "0px 6px 12px -6px rgba(24, 39, 75, 0.12), 0px 8px 24px -4px rgba(24, 39, 75, 0.08)",
     marginRight: theme.spacing(4),
-    [theme.breakpoints.down("sm")]: {
+    [theme.breakpoints.down("755")]: {
```
Comment (Abhishek-kumar09) [inline src/components/GamificationComponent/StatsCardSection.js:70]: Is setting minWidth not enough to give minimum possible width for content to not get cropped?

The breakpoint is hardcoded, just not looking good to me. Is there any other workaround?

**Comment by abhishek2x** [comment]
Hey, I have removed the hardcoded breakpoints, instead, I've used flex property to display the cards.
Screenshots of the same are attached below:

### 1444px

image

### 875px

image

### 555 px

image

Please let me know in case any changes or modifications are required :)

**Review by adarsh-technocrat — APPROVED** [review]
LGTM👍

**Comment by Abhishek-kumar09** [comment]
Thanks @abhishek2x

**Pull request merged**